classes/class-wc-connect-paypal-ec.php

```diff
@@ -68,6 +69,40 @@ public function init() {
 			}
 		}
 
+		public function register_refund_pointer( $pointers ) {
+			$target = '.refund-actions > button:first-child';
+			$link_url = wc_gateway_ppec()->ips->get_signup_url( wc_gateway_ppec()->settings->environment );
+			$pointers[] = array(
+				'id' => 'wc_services_refund_via_ppec',
+				'target' => $target,
+				'options' => array(
+					'content' => sprintf( '<h3>%s</h3><p>%s</p>',
+						__( 'Link a PayPal account' ,'woocommerce-services' ),
+						sprintf( __( "To issue refunds via PayPal Express Checkout, you will need to <a href=\"%s\">link a PayPal account</a> with the email address that received this payment.", 'woocommerce-services' ), $link_url )
```

Comment:
Use single quotes for this string, and pass the result through `wp_kses()` for safety. Example: https://github.com/Automattic/woocommerce-services/blob/367a47accc80b22d28e87df1cd87df7c106a236a/classes/class-wc-connect-help-view.php#L266-L272